micro-rdk/src/esp32/dtls.rs

```diff
@@ -228,7 +231,7 @@ extern "C" fn mbedtls_timing_dtls_set_delay<S>(
     } else if let Some(ssl_ctx) = ctx.ssl_ctx {
         let (_, cx) = unsafe {
             // TODO(RSDK-10196): The `p_bio` field doesn't exist in ESP-IDF 5. Maybe it should be `private_p_bio`?
-            let state = SSLStreamInner::<S>::from_raw((*ssl_ctx).p_bio);
+            let state = SSLStreamInner::<S>::from_raw((*ssl_ctx).private_p_bio);
```

Comment:
we shouldn't do that. We could use a user_data field in the ssl context struct alternatively (it is private as well) however it's accessor functions are inline and therefore not exported by bindgen

Reply:
bindgen doesn't export inlined functions. MbedTLS discourage accessing private field directly I think they just want some stable API